ow_plexil/src/plans/FaultHandlingPattern4.plp

```diff
@@ -0,0 +1,81 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// If no faults are occurring, this plan will pan 180 degrees, 
+// and then unstow and stow the arm.
+// If any fault is injected, the plan will pause and wait for resolution.
+// If a fault is injected during an operation, that operation will only
+// halt immediately if the fault injected is associated with it directly,
+// i.e. an antenna fault during panning or an arm fault during 
+// unstowing or stowing.
+// Otherwise, the pausing will visually become apparent when the next
+// operation does not begin due to it.
+
+#include "plan-interface.h"
+
+LibraryAction InitializeAntennaAndArm ();
+
+FaultHandlingPattern4:
+{
+
+  Boolean PanDone = false;
+  Boolean UnstowDone = false;
+  Boolean StowDone = false;
+
+  Real AngleToGoal;
+
+  LibraryCall InitializeAntennaAndArm ();
+
+  log_info ("Starting FaultHandlingPattern4 plan...");
+
+  Mission: 
+  {
+    
+    PanAntenna:
+    {
+      Repeat !PanDone;
+      Start !Lookup(AntennaFault);
+      Skip PanDone;
+
+      LibraryCall Pan (Degrees=180);
+      
+      AngleToGoal = 180 - abs(Lookup(PanDegrees));
+
+      if abs(AngleToGoal) < 1
+      {
+        PanDone = true;
+      }
```

Comment:
I think AngleToGoal could be negative, which I'm assuming wasn't intended.  The idea of checking the result against a tolerance may be more easily conveyed more compactly:

Real PanToleranceDegrees = 1.0;
...
PanDone = abs(180 - Lookup(PanDegrees)) <= PanToleranceDegrees;

Reply:
Isn't this missing an abs()? PanDone = abs(180 - **abs(**Lookup(PanDegrees)**)**) <= PanToleranceDegrees. I think without it we could end up with an issue if PanDegrees is negative.

Reply:
I don't know the range of PanDegrees offhand -- we should document it somewhere if needed.  But the extra abs seems a good precaution.